Statistics.Test/StatsUnitTest.cs

```diff
@@ -11,20 +12,22 @@ public void ReportsAverageMinMax()
         {
             var statsComputer = new StatsComputer();
             var computedStats = statsComputer.CalculateStatistics(
-                new List<___>{1.5, 8.9, 3.2, 4.5});
+                new List<float>{1.5F, 8.9F, 3.2F, 4.5F});
             float epsilon = 0.001F;
-            Assert.True(Math.Abs(statsComputer.average - 4.525) <= epsilon);
-            Assert.True(Math.Abs(statsComputer.max - 8.9) <= epsilon);
-            Assert.True(Math.Abs(statsComputer.min - 1.5) <= epsilon);
+            Assert.True(Math.Abs(computedStats.average - 4.525) <= epsilon);
+            Assert.True(Math.Abs(computedStats.max - 8.9) <= epsilon);
+            Assert.True(Math.Abs(computedStats.min - 1.5) <= epsilon);
         }
+        [Fact]
         public void ReportsNaNForEmptyInput()
         {
             var statsComputer = new StatsComputer();
             var computedStats = statsComputer.CalculateStatistics(
-                new List<___>{});
-            //All fields of computedStats (average, max, min) must be
-            //Double.NaN (not-a-number), as described in
-            //https://docs.microsoft.com/en-us/dotnet/api/system.double.nan?view=netcore-3.1
+                new List<float>{});
+            Assert.True(computedStats.average.Equals(float.NaN));
+            Assert.True(computedStats.min.Equals(float.NaN));
+            Assert.True(computedStats.max.Equals(float.NaN));
+                       
```

Comment:
Cleanly designed Asserts 👍 
What if the input itself contains NaNs? Can you write a test for that?